app/views/users/index.html.erb

```diff
@@ -1 +1,14 @@
 <h1>Here is the list of users</h1>
+ <% @users.each do |user| %>
+    <div class= "user-card">
+    
+    <%= image_tag(user.photo, :size => "260x180") %>
+<div class= "user-info">
+    <%= user.name %>
```

Comment:
- [ ] You have done an excellent job, your views are set up correctly, congrats on it! 🎉 I would suggest adding a link to the `user.name` so you can click it and see the user's page, this way it would be easier to navigate the site.